azure-monitor-opentelemetry-distro/README.md

```diff
@@ -28,6 +28,21 @@ Install the Azure Monitor Opentelemetry Distro with [pip][pip]:
 pip install azure-monitor-opentelemetry-distro --pre
 ```
 
+### Usage
+
+You can use `configure_azure_monitor` to set up instrumentation for your app to Azure Monitor. `configure_azure_monitor` supports the following optional arguments:
+
+* connection_string - The [connection string][connection_string_doc] for your Application Insights resource. The connection string will be automatically populated from the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable if not explicitly passed in.
+* service_name = Specifies the service name. 
```

Comment:
Link to this [doc](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service) for service_*

Reply:
done